static-assets/components/cstudio-forms/controls/rte-tinymce5.js

```diff
@@ -388,7 +396,8 @@ CStudioAuthoring.Module.requireModule(
             contextmenu: !this.enableSpellCheck,
 
             menu: {
-              tools: { title: 'Tools', items: 'tinymcespellchecker code acecode wordcount' }
+              tools: { title: 'Tools', items: 'tinymcespellchecker code acecode wordcount' },
+              edit: { title: 'Edit', items: 'undo redo | cut copy paste paste_as_text | selectall | searchreplace' }
```

Comment:
Should `paste_as_text` read `paste_from_word` for this to work? Looks like we updated the plugin name but not the button?
```suggestion
              edit: { title: 'Edit', items: 'undo redo | cut copy paste paste_from_word | selectall | searchreplace' }
```

Reply:
`paste_as_text` is a button to paste from a document as plain text. In the plugin we include this:

```
    editor.ui.registry.addToggleButton(PLUGIN_PASTE_TEXT_LABEL, {
      active: false,
      icon: PASTE_TEXT_ICON,
      tooltip: PASTE_TEXT_TOOLTIP,
      onAction: function (api) {
        pasteAsTextOption = !pasteAsTextOption;
        api.setActive(pasteAsTextOption);
        editor.focus();
      },
      onSetup: function (api) {
        editor.on('PastePlainTextToggle', function (e) {
          api.setActive(e.state);
          editor.focus();
        });
      }
    });
```
The name can be anything, but my option is that `paste_as_text` is more explained what the button is about.